llmware/library.py

```diff
@@ -572,7 +840,13 @@ def pull_files_from_cloud_bucket (self, aws_access_key=None, aws_secret_key=None
 
     def generate_knowledge_graph(self):
 
-        """ Builds a statistical co-occurrence matrix for a library """
+        """ Builds a statistical co-occurrence matrix for a library 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Builds a statistical co-occurrence matrix for a library.
```